src/Bridge/Laminas/Module.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Cocur\Slugify\Bridge\Laminas;
+
+use Cocur\Slugify\Slugify;
+use Laminas\ModuleManager\Feature\ServiceProviderInterface;
+use Laminas\ModuleManager\Feature\ViewHelperProviderInterface;
+use Zend\ServiceManager\Config;
+
+/**
+ * Class Module
+ * @package    cocur/slugify
+ * @subpackage bridge
+ * @license    http://www.opensource.org/licenses/MIT The MIT License
+ */
+class Module implements ServiceProviderInterface, ViewHelperProviderInterface
```

Comment:
If you want to make an update of the ZF / Laminas bridge then the support should also be extended for Mezzio (formerly Expressive).
This means to create a module class to support a MVC based application and to create a config-provider class to support a Mezzio based application.

Examples can be found in [laminas-db](https://docs.laminas.dev/laminas-db/):

* [Module class](https://github.com/laminas/laminas-db/blob/2.11.2/src/Module.php)
* [ConfigProvider class](https://github.com/laminas/laminas-db/blob/2.11.2/src/ConfigProvider.php)

As you can see, the module class consumes the configuration from config-provider class.